modules/shared/src/main/scala/org/tessellation/schema/height.scala

```diff
@@ -29,16 +29,18 @@ object height {
     }
   }
 
-  @derive(encoder, decoder, show, eqv)
+  @derive(encoder, decoder, eqv, show)
   @newtype
   case class SubHeight(value: NonNegLong)
 
   object SubHeight {
     val MinValue: SubHeight = SubHeight(NonNegLong.MinValue)
 
+    implicit val order: Order[SubHeight] = Order[Long].contramap(_.value.value)
```

Comment:
It's not related to this PR, I think we shouldn't mix in stuff from outside of the scope in general. If we made a bug by doing that finding it would be a bit harder.

Reply:
it was required when I was changing the code. after all changes it looks like unused indeed. removed